Assets/Scripts/UI/Menu's/TimeScaleController.cs

```diff
@@ -0,0 +1,8 @@
+using UnityEngine;
+
+public static class TimeScaleController
+{
+    public static void SetTimeScaleOnActive() => Time.timeScale = 1;
+
+    public static void SetTimeScaleOnInactive() => Time.timeScale = 0;
```

Comment:
```suggestion
    public static void SetTimeScale(int targetTime) => Time.timeScale targetTime;
```
Je code was niet helemaal D.R.Y.